src/mysql.c

```diff
@@ -973,6 +977,31 @@ static int mysql_read (user_data_t *ud)
 				counter_submit ("mysql_innodb_rows", "read", val, db);
 			else if (strcmp (key, "Innodb_rows_updated") == 0)
 				counter_submit ("mysql_innodb_rows", "updated", val, db);
+
+ 			/* checkpoints */
+ 			else if (strcmp (key, "Innodb_checkpoint_age") == 0)
+ 				gauge_submit ("mysql_innodb_checkpoint", "age", val, db);
+ 			else if (strcmp (key, "Innodb_checkpoint_max_age") == 0)
+ 				gauge_submit ("mysql_innodb_checkpoint", "max_age", val, db);
+ 
+ 			/* max id */
+ 			else if (strcmp (key, "Innodb_max_trx_id") == 0)
+ 				gauge_submit ("mysql_innodb_max", "trx_id", val, db);
+ 
+ 			/* page_size */
+ 			else if (strcmp (key, "Innodb_page_size") == 0)
+ 				gauge_submit ("mysql_innodb_page", "size", val, db);
+ 
+ 			/* open tables and files */
+ 			else if (strcmp (key, "Innodb_open_files") == 0)
+ 				gauge_submit ("mysql_innodb_open", "files", val, db);
+ 			else if (strcmp (key, "Innodb_open_tables") == 0)
+ 				gauge_submit ("mysql_innodb_open", "tables", val, db);
```

Comment:
@octo, What will happen when we, for example, will collect 'Innodb_open_files' (fictional variable) ? 
By this logic, that should be 'objects' type and 'files-open' type_instance. But what will be wrong.

I propose to use something like this: 'mysql_innodb_tables' type (if it does not exists, please check!) and 'open' type instance. Then there no conflict appear in future. 

What do you think?